torch/_inductor/codegen/triton.py

```diff
@@ -818,6 +819,7 @@ def __init__(
         pid_cache=None,
         reduction_hint=ReductionHint.DEFAULT,
         min_elem_per_thread=0,
+        deviceOverrides=CUDADeviceOverrides
```

Comment:
We added multiple `DeviceOverrides`. Let's add it to the `Virtualized` and set `DeviceOverrides` according to the graph information.